app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Help</h1>
+<p>
+#Get help on the Ruby on Rails Tutorial at the
+link_to
+=> <a href="http://www.railstutorial.org/help">Rails Tutorial help page</a>.
```

Comment:
use link_to instead of a tag

Reply:
why my file did not updated from sublime?

Reply:
ready